<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite start="1438694906540" stop="1438694909662" version="1.4.15" xmlns:ns2="urn:model.allure.qatools.yandex.ru">
    <name>Parametrized Tests</name>
    <title>Parametrized Tests</title>
    <test-cases>
        <test-case start="1438694907945" stop="1438694908070" status="passed">
            <name>UNKNOWN_ID</name>
            <title>UNKNOWN_ID[UNKNOWN_ID,1,String]</title>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="IE00115341"/>
                <label name="thread" value="Parametrized Tests.1"/>
            </labels>
            <parameters>
                <parameter name="Test ID" value="UNKNOWN_ID" kind="argument"/>
                <parameter name="#" value="1" kind="argument"/>
                <parameter name="Expected value" value="String" kind="argument"/>
            </parameters>
        </test-case>
        <test-case start="1438694908539" stop="1438694908540" status="passed">
            <name>WARNING: EMPTY TEST ID</name>
            <title>WARNING: EMPTY TEST ID[,2,]</title>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="IE00115341"/>
                <label name="thread" value="Parametrized Tests.1"/>
            </labels>
            <parameters>
                <parameter name="Test ID" value="" kind="argument"/>
                <parameter name="#" value="2" kind="argument"/>
                <parameter name="Expected value" value="" kind="argument"/>
            </parameters>
        </test-case>
        <test-case start="1438694908856" stop="1438694908857" status="passed">
            <name>TORF-37961_Func-2</name>
            <title>UPDATED_Verify that an error message is displayed after an unsuccessful login attempt[TORF-37961_Func-2,3,Long text here very long text here very lo...]</title>
            <description type="text">Test case verifies if an error message is displayed after an unsuccessful login attempt</description>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="IE00115341"/>
                <label name="thread" value="Parametrized Tests.1"/>
                <label name="story" value="TORF-10397: As a TOR SSO user, I want an alternative authentication solution so that I can log into the TOR launcher when TOR SSO is uninstalled or down"/>
                <label name="group" value="KGB"/>
                <label name="group" value="RFA"/>
                <label name="feature" value="Login Page"/>
                <label name="severity" value="normal"/>
                <label name="execution_type" value="automated"/>
                <label name="testId" value="TORF-37961_Func-2"/>
            </labels>
            <parameters>
                <parameter name="Test ID" value="TORF-37961_Func-2" kind="argument"/>
                <parameter name="#" value="3" kind="argument"/>
                <parameter name="Expected value" value="Long text here very long text here very long text here very long text here very long text here very long text here very long text here very long text here very long text here very long text here very long text here very" kind="argument"/>
            </parameters>
        </test-case>
        <test-case start="1438694909238" stop="1438694909239" status="passed">
            <name>TORF-37961_Func-2</name>
            <title>UPDATED_Verify that an error message is displayed after an unsuccessful login attempt[TORF-37961_Func-2,4,Something]</title>
            <description type="text">Test case verifies if an error message is displayed after an unsuccessful login attempt</description>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="IE00115341"/>
                <label name="thread" value="Parametrized Tests.1"/>
                <label name="story" value="TORF-10397: As a TOR SSO user, I want an alternative authentication solution so that I can log into the TOR launcher when TOR SSO is uninstalled or down"/>
                <label name="group" value="KGB"/>
                <label name="group" value="RFA"/>
                <label name="feature" value="Login Page"/>
                <label name="severity" value="normal"/>
                <label name="execution_type" value="automated"/>
                <label name="testId" value="TORF-37961_Func-2"/>
            </labels>
            <parameters>
                <parameter name="Test ID" value="TORF-37961_Func-2" kind="argument"/>
                <parameter name="#" value="4" kind="argument"/>
                <parameter name="Expected value" value="Something" kind="argument"/>
            </parameters>
        </test-case>
        <test-case start="1438694909639" stop="1438694909640" status="passed">
            <name>TORF-37961_Func-2</name>
            <title>UPDATED_Verify that an error message is displayed after an unsuccessful login attempt[TORF-37961_Func-2,5,Some Value]</title>
            <description type="text">Test case verifies if an error message is displayed after an unsuccessful login attempt</description>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="IE00115341"/>
                <label name="thread" value="Parametrized Tests.1"/>
                <label name="story" value="TORF-10397: As a TOR SSO user, I want an alternative authentication solution so that I can log into the TOR launcher when TOR SSO is uninstalled or down"/>
                <label name="group" value="RFA"/>
                <label name="group" value="KGB"/>
                <label name="feature" value="Login Page"/>
                <label name="severity" value="normal"/>
                <label name="execution_type" value="automated"/>
                <label name="testId" value="TORF-37961_Func-2"/>
            </labels>
            <parameters>
                <parameter name="Test ID" value="TORF-37961_Func-2" kind="argument"/>
                <parameter name="#" value="5" kind="argument"/>
                <parameter name="Expected value" value="Some Value" kind="argument"/>
            </parameters>
        </test-case>
        <test-case start="1412949549377" stop="1412949551163" status="failed">
            <name>calculationTest[299]</name>
            <failure>
                <message>AssertionError: Expected 9 less than 8 </message>
                <stack-trace>java.lang.AssertionError: Expected 9 less than 8
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at my.company.ParametrizedTest.calculationTest(ParametrizedTest.java:56)
	at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runners.Suite.runChild(Suite.java:127)
	at org.junit.runners.Suite.runChild(Suite.java:26)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)
	at java.util.concurrent.FutureTask.run(FutureTask.java:166)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:722)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="story" value="ParametrizedStory"/>
                <label name="feature" value="SimpleStories"/>
                <label name="host" value="my.cool.host.com"/>
                <label name="thread" value="pool-1-thread-16"/>
                <label name="severity" value="normal"/>
                <label name="execution_type" value="automated"/>
            </labels>
            <parameters>
                <parameter name="Argument" value="299" kind="environment-variable"/>
            </parameters>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TAF"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
